fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/listitem/ListItem.kt

```diff
@@ -285,14 +285,16 @@ object ListItem {
             Alignment.Bottom -> Alignment.BottomEnd
             else -> Alignment.CenterEnd
         }
+        val isClickable = onClick != null
+
         Row(
             modifier
                 .background(backgroundColor)
                 .fillMaxWidth()
                 .height(IntrinsicSize.Max)
                 .borderModifier(border, borderColor, borderSize, borderInsetToPx)
                 .clickAndSemanticsModifier(
-                    interactionSource, onClick = onClick ?: {}, enabled, rippleColor
+                    interactionSource, onClick = onClick ?: {}, enabled=isClickable, rippleColor
```

Comment:
nit: lint space after before '='

Reply:
enabled should not be updated here. it is taken from the parameter